### ISSUE TITLE
Exclude modules from obfuscation

### ISSUE THREAD

**Issue by zfoot** [rockbruno/swiftshield#43]
The parameters for obfuscating certain modules in automatic mode, if I wanted to exclude modules from cocoapod frameworks(e.g. RxSwift), I will input the parameters this way: "-ignore-modules RxSwift"? Because I tried it on certain modules however they're still obfuscated.

P.S: If it's possible, I wanted to exclude all cocoapod modules from being obfuscated, is there a shortcut for doing so?(e.g. "-ignore-modules pods")

**Comment by rockbruno** [comment]
That's right. For multiple modules, you can use a comma: `-ignore-modules RxSwift,Cartography`. The only requirement is that the name you use for the module is not the pod name, but the name used in import statements.

There's no shortcut for ignoring all pods, but it's something that could be implemented.
